nixos/modules/services/web-servers/h2o/common.nix

```diff
@@ -0,0 +1,27 @@
+{ lib }:
+{
+  tlsRecommendationsOption = lib.mkOption {
+    type = lib.types.nullOr (
+      lib.types.enum [
+        "modern"
+        "intermediate"
+        "old"
+      ]
+    );
+    default = null;
+    example = "intermediate";
+    description = ''
```

Comment:
Could you document what `null` does? "If not set, ..."

Reply:
If I were coming to this module knowing nothing about h2o, and just a basic understanding of http and https, I'd be wondering things like:

- "How dangerous is the default behavior here?"
- "Why isn't one of these chosen for me?"
- "Why am I not being forced to pick one of these?"